scripts/cr.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+REQ_FILES=(
+  "/app/requirements"
+  "/app/requirements.dev"
+)
+
+for f in "${REQ_FILES[@]}"; do
+  rm ${f}.txt && pip-compile --generate-hashes --resolver=backtracking -o ${f}.txt ${f}.in || exit 1;
```

Comment:
The `rm` command will fail if, for whatever reason (accident?), the current 'requirements' file doesn't already exist, and then the whole script will exit. We would not be able to compile a requirements file from scratch with this script, unless you trick the script by creating an empty requirements file first.

Is the `rm` command necessary? The pip-tools documentation suggest that `pip-compile` would behave sensibly if it found an existing requirements.txt file.